scripts/apps/authoring/views/full-preview-item.html

```diff
@@ -146,6 +146,10 @@
             <p><i class="icon-picture" ng-if="hideMedia"></i>{{item.associations.featuremedia.description_text}}</p>
         </div>
 
+        <div ng-if="!hideMedia && item.type === 'picture' && item.renditions.original.href" style="margin-bottom: 20px">
+            <img ng-src="{{item.renditions.original.href}}" />
```

Comment:
not sure we should use the original rendition, it's huge file usually for ansa, baseImage might be enough (maybe we can have it configurable)